### ISSUE TITLE
Prevent villagers with different professions from stacking

### ISSUE THREAD

**Issue by CW-UK** [Nathat23/StackMob-5#103]
I can't seem to stop villagers with different professions from stacking. Example:

1. Spawn two villagers apart from each other (outside the merge radius)
2. Give both a different workstation so that they gain professions
3. Let the two villagers meet

Both villagers will merge into one, despite having two different professions. I've tried some config options such as:

  profession: true
  villager-profession: true
  fletcher: true
  profession-fletcher: true

**Comment by Aurelien30000** [comment]
Good idea, but villagers shouldn't be authorized to stack themselves, just add them to type_blacklist ^^

**Comment by CW-UK** [comment]
Forgive my ignorance for asking, but I'm going by the comment in the config file:

`# Whether entity specific traits (e.g. profession, colour) should be segregated (*)`

I'm aiming for villagers of different professions to stack, which seems to be suggested by this comment.

**Comment by Aurelien30000** [comment]
Currently there is no option for villagers, maybe we could add one ? :p

**Comment by CW-UK** [comment]
I believe it's a valid addition, as villagers can stack up in their professions from spawners and be cycled through by killing them off. I know other networks that use StackMob that stack their villagers by profession, but they are using an older version, so maybe it's been removed in later versions? Or perhaps they modified the plugin.

In the mean time it may save people like from me from wasting their time if the config line is adjusted so that it doesn't indicate profession segregation is possible.

**Comment by Aurelien30000** [comment]
@CW-UK I added this to the latest pull request's build -> https://github.com/Nathat23/StackMob-5/pull/101 

WARNING : Contains others huge changes that could cause issues currently, be aware and don't hesitate to report issues ^^
Thanks!

**Comment by Nathat23** [comment]
I think I never added support for villager professions because there's no way to keep the trades that villagers may have.

**Comment by Aurelien30000** [comment]
> I think I never added support for villager professions because there's no way to keep the trades that villagers may have.

Exactly, but it doesn't costs anything to have it :)
